lib/Marlin/Marlin/src/feature/prusa/MMU2/protocol_logic.cpp

```diff
@@ -736,7 +736,11 @@ void ProtocolLogic::FormatLastResponseMsgAndClearLRB(char *dst) {
 }
 
 void ProtocolLogic::LogRequestMsg(const uint8_t *txbuff, uint8_t size) {
-    constexpr uint_fast8_t rqs = modules::protocol::Protocol::MaxRequestSize() + 1;
+    constexpr unsigned char mrs = modules::protocol::Protocol::MaxRequestSize();
```

Comment:
If you investigate the code, you'll find out that the size of the requests is rather fixed, so it won't exceed the maximum size by design and these checks are not necessary. An assert would be more fitting here.

Reply:
thx, added an assert :)